docs/tutorials/Hosting_on_Heroku.md

```diff
@@ -0,0 +1,197 @@
+
+## Connecting Heroku to GitHub
+
+In this tutorial, we'll use a Github repository as the source for the Heroku app. Before you can deploy, you should [push your project to Github](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository).
+
+> INFO
+> If you prefer to deploy from your local app, you can [use the Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
+
+After you have your GitHub repo and Heroku application set up, you will be met with some options in the **Deploy** tab. From here, you have several deployment method options to choose from
+
+![Heroku dashboard](heroku-connectGH.png)
+
+Click **Connect this app to GitHub**.
+ 
+> INFO
+> that you can only connect Heroku apps to a single GitHub repository. 
+
+From there, you will be able to manage the app's deployment method:
+
+![Heroku deployment](heroku-deploy.png)
+
+Selecting a branch and enabling **Automatic Deploys** for your app will deploy a new version of this app every time a push occurs. **Manual Deploys** would require you to select a specific branch before updates go live for your bot. 
+
+> INFO
+> If you want to do this with the Heroku CLI, there are [separate steps in the dashboard](https://devcenter.heroku.com/articles/heroku-cli) you can follow to accomplish this.
+
+## Configuring app in Heroku
+
+Before your app can go online, you'll have to configure your Heroku environment with your Discord bot's credentials.
+
+### Adding app credentials
+
+In the **Settings** tab, add your bot’s `TOKEN`, `GUILD_ID`, `CLIENT_ID` from [the section above](#DOCS_TUTORIALS/hosting-on-heroku#creating-your-discord-app), and any other tokens or API keys you want for your own app.
+
+> INFO
+> If you're developing your app locally, you can store your credentials in a `.env` file (see `.sample.env` for an example)
+
+![Configuring variables in Heroku](heroku-configVars.png)
+
+You'll notice that there isn't anything you have to change in your source code. Amazin'
+
+### Adding a buildpack and configuring a dyno
+
+The next step is to **add a buildpack to your app**. The buildpack is responsible for compiling your deployed code and installing any dependencies, meaning the one you select will also depend on how you built your bot. Since Heroku-Baker is a node app, we'll need to add the nodeJS buildpack.
+
+![Adding a buildpack](heroku-buildpack.png)
+
+Once you successfully connect your app to your bot's repo, you'll notice a new window in the dashboard called **Dyno formation**.
+
+A **Dyno** is a virtualized Linux container that executes the code in `Procfile` - think of it as a mini computer dedicated to running your bot. By default, Heroku gives 550 free dyno hours per application per month. [This is just enough to keep your bot online 24/7] If your app will need additional hours, you can look at Heroku's [pricing page](https://www.heroku.com/pricing).
```

Comment:
This isn't enough actually, a month has ~730 hours. Also, if the user adds a credit card (without buying anything), Heroku gives more 450 hours (1k hours total). [Refs](https://devcenter.heroku.com/articles/free-dyno-hours#free-dyno-hour-pool)